rfcs/52-unstable_features.md

```diff
@@ -0,0 +1,89 @@
+
+This would also improve the contributor experience for PRs that stay open for some time and ends up conflicting with main.
+
+In this RFC, "feature" means the new feature or change under development, and "feature gate" the conditional compilation flag. "unstable" means a feature that is subject to change, not something that will break Bevy.
+
+## User-facing explanation
+
+So you want to contribute a new feature to Bevy! You have a few paths available to you, depending on the scope of the feature:
+- A small feature with a clear implementation? Go ahead and open that PR!
+- A large feature with an open design space that could use some discussion about how to implement it? A RFC would be good.
+- Something in between that would benefit from iterative work? Here come the unstable feature workflow for you!
+
+You will need to open a tracking issue to explain the new feature you want to add. If it looks good, you'll be ready to open that PR with a feature gate for your changes!
+
+## Implementation strategy
+
+The new workflow for this kind of feature would be:
+- Submit an issue explaining the feature you want to add under a feature gate. This will be the tracking issue for stabilization once approved. This issue gets a new label `S-Stabilization`.
+- If approved, submit a PR with the new feature and the feature gate
+- This PR can be merged by someone with the merge rights on the related part of Bevy as soon as it's `S-Ready-For-Final-Review`, without an approval by the main Bevy maintainers.
+- Subsequent PRs on the feature must be linked to the tracking issue and follow the same process.
+- Once the feature is finished, enter the stabilization period. Open a PR that will remove the feature gate. This PR needs to be approved by the main Bevy maintainers.
+
+The feature gate should follow the following convention for its name: `unstable-<#tracking-issue-number>-<feature-name>`.
+
+An existing PR can be retroffited in this process on the suggestion of Bevy community members, and if the PR author agrees.
+
+The RFCs and the unstable features are closely related. An RFC can be implemented as an unstable feature to help resolve issues that need more experimentation, or split a large implementation. An unstable feature could require an RFC to reach stabilization. For example:
+- RFC to Unstable Feature
+    - Open a RFC, reaching a design
+    - Implement part of the RFC as an unstable feature
+    - Feedback into the RFC with the gained knowledge of implementation and usage
+- Unstable Feature to RFC
+    - Open an tracking issue, start implementing it
+    - Part of the unstable feature is larger than expected with several design possibilities
+    - Open a RFC, reaching a decision on the design
+    - Implement the RFC for the unstable feature
+
+
+Each PR will still need to be approved by Bevy community members, and would still need to have the same level of quality as other PRs merged.
```

Comment:
Review this inline with more recent rules about maintainers and reviewers? The rules is 2 community reviews I think.